roadmap.md

```diff
@@ -79,7 +79,11 @@ After the initial release, WebAssembly has been gaining new features through the
       );
       for (let { name, description, url, phase } of features) {
         let supportHTML = h('td');
-        wasmFeatureDetect[name]()
+        Promise.resolve()
+          // Make sure to call `wasmFeatureDetect` inside a promise-chained
+          // function so that we can still render table rows for features that
+          // we don't have a detector yet.
+          .then(() => wasmFeatureDetect[name]())
```

Comment:
Makes sense. Do you want to submit a PR to add a detector for module linking though?

Reply:
Already filed https://github.com/GoogleChromeLabs/wasm-feature-detect/issues/38 -- it is a little more complicated because WABT doesn't support module linking yet.